src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -245,13 +245,13 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
      * @param resultLines - Input of ExecutionResult object
      * @return map with jobid as key
      */
-    private Map<BEJobID, GenericJobInfo> processQstatOutput(List<String> resultLines) {
+    private Map<BEJobID, GenericJobInfo> processQstatOutput(String result) {
```

Comment:
This is possible because you spit out XML now, right?

Reply:
Yes that's right